R/fixed_regex_linter.R

```diff
@@ -3,7 +3,7 @@
 #' Invoking a regular expression engine is overkill for cases when the search
 #'   pattern only involves static patterns.
 #'
-#' NB: for `stringr` functions, that means wrapping the pattern in `stringr::fixed()`.
```

Comment:
i think lower-case is correct, though apparently it's OK to capitalize sometimes:

https://www.grammarly.com/blog/capitalization-after-colons/#:~:text=Capitalization%3A%20First%20Word%20After%20a,it%20begins%20a%20complete%20sentence.

Reply:
Thanks for checking!

I was mostly motivated by internal consistency here :)

<img width="562" alt="Screenshot 2022-07-31 at 08 16 13" src="https://user-images.githubusercontent.com/11330453/182012926-b348b838-aad9-4240-9df7-b752189ba210.png">

Reply:
indeed! let's change that one, too :)

Reply:
Okay, both of them are now lower-case, and, thus, consistent with each other.